.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,9 +20,6 @@ Please create a second pull request at https://github.com/shopware/docs
 <!-- Examples:
 - closes #123  - closes the issue #123 when the PR is merged
 - relates #123 - relates to the issue #123
-
-In case of issue existing only on Jira, link to the Jira issue.
```

Comment:
that seems not related to the changes :thinking:

Reply:
I deleted that, because AI did used NEXT for PR title 🤣

Reply:
for now, we still need it :grimacing: as we are not yet fully on GitHub yet